scripts/globals/abilities/rune_enchantment.lua

```diff
@@ -0,0 +1,22 @@
+-----------------------------------
+-- Ability: Rune Enchantment
+-- Allows you to harbor runes
+-- Obtained: Rune Fencer Level 5
+-- Recast Time: 0:05
+-- Duration: 5:00
+-----------------------------------
+require("scripts/globals/status")
+require("scripts/globals/msg")
+-----------------------------------
+local ability_object = {}
+
+ability_object.onAbilityCheck = function(player, target, ability)
+    return 0, 0
+end
+
+--this is a dummied ability that is not actually usable by a player due to Rune Enchantment only being the main category in a menu, as well as the name of the ability for recast.
```

Comment:
Usually that sort of thing doesn't need a script

Reply:
Seems to work without it. It was originally just a helper lua with a name that made sense. I can delete it if it's not useful as a form of documentation.